src/main/scala/io/citrine/lolo/bags/BaggerUtil.scala

```diff
@@ -0,0 +1,58 @@
+package io.citrine.lolo.bags
+
+import io.citrine.lolo.util.Async
+import io.citrine.lolo.{Model, PredictionResult}
+
+import scala.collection.parallel.immutable.ParSeq
+
+case class BaggerHelper(
```

Comment:
Docstring?  Should this class be protected?

Reply:
This is only used in the MultiTaskBagger; did you mean to use it in the Bagger too?

Reply:
Documented and protected.
I intend to use this in Bagger too, but I want to do that cleanup in another PR to get this out the door.